### PR TITLE
dont emit references for 'this'

### LINKED ISSUE #155: Stop emitting references to `this`
https://github.com/sourcegraph/lsif-java/blob/687fef86c382ddedd9aaa462df0ef33589a8dded/tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyDiffer.java#L96-L99

```diff
- obtained
+ expected
      this.executor = new HandlerExecutor(handler);
- //  ^^^^ reference com/airbnb/epoxy/AsyncEpoxyDiffer#
  //       ^^^^^^^^ reference com/airbnb/epoxy/AsyncEpoxyDiffer#executor.
  //                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ reference com/airbnb/epoxy/HandlerExecutor#`<init>`().
```

### PR DESCRIPTION
**Pull request by Strum355**
Closes #155

### CODE REVIEW

File: tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java [hunk 1]
```diff
@@ -32,7 +32,6 @@
   public AsyncEpoxyController() {
 //       ^^^^^^^^^^^^^^^^^^^^ definition com/airbnb/epoxy/AsyncEpoxyController#`<init>`(). public AsyncEpoxyController()
     this(true);
-//  ^^^^ reference com/airbnb/epoxy/AsyncEpoxyController#`<init>`(+1).
```
Comment (olafurpg) [inline tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java:35]: Hmm, I think we should continue to emit references to constructors. It's only when the reference points to a type symbol when we want to skip it. IntelliJ support goto def on `this()` when it points to a constructor.

**Review by olafurpg — CHANGES_REQUESTED** [review]

File: tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java (hunk 1 above)
Comment (Strum355) [reply tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java:35]: Good catch! Easy to miss this amongst the noise 🙂

Comment (Strum355) [reply tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java]: Fixed :+1:

File: semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java [hunk 2]
```diff
@@ -24,6 +25,7 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import static com.sourcegraph.semanticdb_javac.Debugging.pprint;
```
Comment (olafurpg) [inline semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java:28]: Does import?

**Review by olafurpg — APPROVED** [review]
LGTM 🚀

File: semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java (hunk 2 above)
Comment (Strum355) [reply semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java:28]: whoops, forgot to remove

**Pull request merged**